tests/integration/suite/daprd/pubsub/grpc/errors.go

```diff
@@ -190,7 +189,7 @@ func (e *standardizedErrors) Run(t *testing.T, ctx context.Context) {
 			}
 		}
 		require.NotNil(t, errInfo, "ErrorInfo should be present")
-		require.Equal(t, kiterrors.CodePrefixPubSub+apierrors.PostFixNameEmpty, errInfo.GetReason())
```

Comment:
if you dont want the concatenation of the consts, Id rather a file dedicated to these consts over magic strings.

Reply:
In the context of tests, I would cast my vote for strings. Since this is as an api exposed to users, they rely on that specific string, so that's why comparing to it directly is safer.

Reply:
Indeed :+1